runtime-modules/proposals/codex/src/proposal_types/mod.rs

```diff
@@ -47,14 +46,8 @@ pub enum ProposalDetails<MintedBalance, CurrencyBalance, BlockNumber, AccountId,
     /// Balance for the `set content working group mint capacity` proposal
     SetContentWorkingGroupMintCapacity(MintedBalance),
 
-    /// AccountId for the `evict storage provider` proposal
-    EvictStorageProvider(AccountId),
```

Comment:
I just realized we actually need to keep a "dummy" value here to preserve the enumeration index correct when reading state of proposals of this type. 

can we change it to:
`_EvicStorageProvider(AccountId)`

Reply:
Fixed.